### PR TITLE
relax for prettier

### PR DESCRIPTION
**Pull request by sballesteros**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,60 +1,43 @@
...
-    'no-use-before-define': 0,
-    'brace-style': 0,
-    'key-spacing': ['error', { beforeColon: false, afterColon: true, mode: 'minimum' }],
-    'no-multi-spaces': 0,
-    'comma-dangle': 0,
-    'no-confusing-arrow': 0,
-    'template-curly-spacing': 0,
-    'no-plusplus': 0,
-    'jsx-a11y/img-redundant-alt': 0,
+    'no-console': 0,
```
Comment (darobin) [inline index.js]: For this I find that `'no-console': ['warn', { allow: ['error', 'warn'] }],` is pretty good at catching debugging code left hanging around.

Comment (sballesteros) [inline index.js]: thanks will add

File: index.js [hunk 2]
```diff
@@ -1,60 +1,43 @@
...
   rules: {
-    'arrow-parens': 0,
-    'consistent-return': 0,
-    'no-param-reassign': 0,
-    'no-continue': 0,
-    'no-return-assign': 0,
-    'no-underscore-dangle': 0,
-    'prefer-spread': 0,
-    'class-methods-use-this': 0,
-    'no-constant-condition': ['error', { checkLoops: false }],
```
Comment (darobin) [inline index.js]: This one is useful for `while (true)` support.

File: index.js [hunk 3]
```diff
@@ -1,60 +1,43 @@
...
-    'no-shadow': ['error', { allow: ['err', 'error', 'res', 'resp', 'body'] }],
-    'no-bitwise': ['error', { allow: ['~'] }],
```
Comment (darobin) [inline index.js]: This one catches type `&` instead of `&&` and friends.

File: index.js [hunk 4]
```diff
@@ -1,60 +1,43 @@
...
-    'no-cond-assign': ['error', 'except-parens'],
```
Comment (darobin) [inline index.js]: This one makes your `createError()` thing possible with parens.

File: index.js [hunk 5]
```diff
@@ -1,60 +1,43 @@
...
-    'react/sort-comp': 0,
-    'react/jsx-no-target-blank': 0,
-    'react/jsx-filename-extension': 0,
-    'space-before-function-paren': 0,
-    'one-var': 0,
-    // in practice, people indent responsibly - this rule is too inflexible for many justified cases
-    indent: 0,
-    quotes: ['error', 'single', { avoidEscape: true, allowTemplateLiterals: true }],
-    'comma-style': ['error', 'last', { exceptions: { VariableDeclaration: true } }],
-    'no-restricted-syntax': ['error', 'WithStatement'],
```
Comment (darobin) [inline index.js]: This one blows up on `with` statements.

File: index.js [hunk 6]
```diff
@@ -1,60 +1,43 @@
...
-    'no-use-before-define': 0,
-    'brace-style': 0,
-    'key-spacing': ['error', { beforeColon: false, afterColon: true, mode: 'minimum' }],
-    'no-multi-spaces': 0,
-    'comma-dangle': 0,
-    'no-confusing-arrow': 0,
-    'template-curly-spacing': 0,
-    'no-plusplus': 0,
-    'jsx-a11y/img-redundant-alt': 0,
```
Comment (darobin) [inline index.js]: This one is well-meaning, but it triggers in wrong cases — we probably need to keep it.

File: index.js (hunk 2 above)
Comment (sballesteros) [inline index.js]: added

File: index.js (hunk 3 above)
Comment (sballesteros) [inline index.js]: added

File: index.js (hunk 4 above)
Comment (sballesteros) [inline index.js]: added

File: index.js (hunk 5 above)
Comment (sballesteros) [inline index.js]: added

File: index.js (hunk 6 above)
Comment (sballesteros) [inline index.js]: added

**Pull request merged**